exist-core/src/test/xquery/xquery3/higher-order.xml

```diff
@@ -475,12 +475,7 @@ return
         <expected>23 29</expected>
     </test>
     <test output="text">
-        <task>fn:filter function (wrong argument order: deprecated)</task>
-        <code>fn:filter(function($a) {$a mod 2 = 0}, 1 to 10)</code>
-        <expected>2 4 6 8 10</expected>
-    </test>
-    <test output="text">
-        <task>fn:filter function (correct argument order)</task>
+        <task>fn:filter function</task>
```

Comment:
Maybe don't remove the test - but change it so that it is expected to fail. This proves then that your changes are complete :-)

Reply:
OK, will do.